src/frontend/src/pages/GanttPage/GanttPageFilter.tsx

```diff
@@ -0,0 +1,197 @@
+            <MenuItem value="All Statuses">All Statuses</MenuItem>
+            {Object.values(WbsElementStatus).map((status) => {
+              return <MenuItem value={status}>{status}</MenuItem>;
+            })}
+          </Select>
+        </FormControl>
+      </Grid>
+      <Grid item xs={12} md={2}>
+        <FormControl sx={{ width: '100%' }}>
+          <FormLabel>Team</FormLabel>
+          <Select value={selectedTeam} onChange={teamHandler}>
+            <MenuItem value="All Teams">All Teams</MenuItem>
+            {teamList.map((team) => (
+              <MenuItem value={team}>{team}</MenuItem>
+            ))}
+          </Select>
+        </FormControl>
+      </Grid>
+      <Grid item xs={12} md={2}>
+        <FormControl sx={{ width: '100%' }}>
+          <FormLabel>Start Date</FormLabel>
+          <DatePicker
+            inputFormat="yyyy-MM-dd"
+            onChange={startHandler}
+            className={'padding: 10'}
+            value={currentStart}
+            renderInput={(params) => <TextField autoComplete="off" {...params} />}
+          />
+        </FormControl>
+      </Grid>
+      <Grid item xs={12} md={2}>
+        <FormControl sx={{ width: '100%' }}>
+          <FormLabel>End Date</FormLabel>
+          <DatePicker
+            inputFormat="yyyy-MM-dd"
+            onChange={endHandler}
+            className={'padding: 10'}
+            value={currentEnd}
+            renderInput={(params) => <TextField autoComplete="off" {...params} />}
+          />
+        </FormControl>
+      </Grid>
+      <Grid
+        item
+        container
+        xs={12}
+        md={3}
+        spacing={1}
+        sx={{ justifyContent: 'end', alignItems: 'center', alignSelf: 'center', justifySelf: 'end' }}
+      >
+        <Grid item>
+          <Button
+            variant="contained"
+            onClick={() => {
+              expandedHandler(true);
+            }}
+            startIcon={<UnfoldMoreIcon />}
+            sx={{
+              '&:hover': {
+                backgroundColor: '#ff0000'
+              }
+            }}
+          >
+            Expand
+          </Button>
+        </Grid>
+        <Grid item>
+          <Button
+            variant="contained"
+            onClick={() => {
+              expandedHandler(false);
+            }}
+            startIcon={<UnfoldLessIcon />}
+            sx={{
+              '&:hover': {
+                backgroundColor: '#ff0000'
+              }
+            }}
+          >
+            Collapse
+          </Button>
+        </Grid>
+        <Grid item>
+          <Button
+            variant="contained"
+            onClick={resetHandler}
+            startIcon={<RestartAltIcon />}
+            sx={{
+              '&:hover': {
+                backgroundColor: '#ff0000'
+              }
+            }}
+          >
+            Reset
+          </Button>
+        </Grid>
+      </Grid>
+    </Grid>
+  </PageBlock>
+);
+
+export default GanttPageFilter;
```

Comment:
Looks really good, you could break out the car section and the button section into helper methods called renderCarFilter, and renderButtons if you wanna improve readability a bit, this file the returned component is starting to get a lil long